### ISSUE TITLE
halite2

### ISSUE THREAD

**Issue by jcjones94** [FiodorG/Halite2#5]
when halite ended did any recruiters contact you?

**Comment by FiodorG** [comment]
No, never. You? Either they only contacted people more at the top, or you have to apply yourself. I was thinking the latter.
